src/components/Home.js

```diff
@@ -169,73 +259,73 @@ const Home = () => {
               responsive={responsive}
               keyBoardControl={true}
               transitionDuration={800}
-              containerClass="carousel-container"
+              containerClass="carousel-icon-container"
               removeArrowOnDeviceType={['mobile']}
               itemClass="carousel-item-padding-40-px"
             >
-              <div className='grocery'>
-                <div className='icon' value='Grocery'>
-                  <i className="ri-shopping-cart-line"></i>
+              <div className='grocery' id='Grocery' onClick={categoryHandler}>
```

Comment:
@tams2429 for each of these <div> components, I think we should be rendering them in a loop rather than hard coding each category, it'll 1.) reduce maintence burden 2.) we can then add new categories to a single data array in one place. See this [article ](https://blog.cloudboost.io/for-loops-in-react-render-no-you-didnt-6c9f4aa73778) for an example on how to render HTML in loops

Reply:
Yep will look at this later after we get this integrated with the backend